cvat-data/src/ts/cvat-data.ts

```diff
@@ -155,17 +155,43 @@ export class FrameDecoder {
         }
     }
 
+    private validateFrameNumbers(frameNumbers: number[]): void {
+        if (!frameNumbers || !frameNumbers.length) {
+            throw new Error('frameNumbers must not be empty');
+        }
+
+        // ensure is ordered
+        for (let i = 1; i < frameNumbers.length; ++i) {
+            const prev = frameNumbers[i - 1];
+            const current = frameNumbers[i];
+            if (current <= prev) {
+                throw new Error(
+                    'frameNumbers must be sorted in ascending order, ' +
+                    `got a (${prev}, ${current}) pair instead`,
+                );
+            }
+        }
+    }
```

Comment:
Is there any reason to make it class method?

Reply:
Just suggestion to move it out of the class as it does not use `this` context somehow. Generally it is a minor question.